src/converters/cover.ts

```diff
@@ -173,6 +185,28 @@ class CoverHandler implements ServiceHandler {
   updateState(state: Record<string, unknown>): void {
     this.monitors.forEach((m) => m.callback(state));
 
+    if ('motor_state' in state) {
```

Comment:
I don't like the idea of just assuming the name of a key if it wasn't specifically in the exposes information.
I've seen errors because of this in the past that we're very hard to track (especially when people cache the state in Zigbee2MQTT and then an update of Z2M changes the exposes information).

I think the `ServiceCreator` in the top of the file should be extended to see if there's an entry with `motor_state` (on the same endpoint).
Then we do not have to "guess" if this property may exist.